### DISCUSSION TITLE
[volume_manuel_valide] en cas de réponses "non-multiples"

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#18] [Q&A] [answered]
@VeloTerritoires

Dans la méthodologie, il est indiqué comment remplir le champ [volume_manuel_valide] dans le cas de questionnaire multiples mais je ne trouve rien pour les cas de questionnaires uniques.

Je suppose que dans ce cas de figure, on reporte la valeur de [volume_manuel].
Par contre, je ne trouve pas de vérification entre les [volume_manuel] (comptage manuel) et [taille_groupe_totale](réponse questionnaire).
Que se passe-t-il s'il y a une différence entre ces 2 valeurs?

**Comment by SteffMangin** [comment, marked as answer]
@JMPivette 
Oui, on reporte le [volume_manuel] dans [volume_manuel_valide]. Et si, il est prévu de faire une vérif de l'info issue du comptage manuel et des enquêtes pour ceux ayant répondu bien sûr.
Si je ne me trompe pas dans la version de la méthode que tu as, cela doit être le paragraphe 3.1.10 Vérification des volumes de personnes selon les bases.
En cas de différence conséquente, il est dit qu'il faut en chercher la raison. Pour toi, cela induit qu'il doit y avoir une alerte mais pas de traitement. C'est à un humain d'intervenir.

Est-ce que cela répond à ta question ?

**Reply by JMPivette** [reply]
Merci 👍
